crates/provider-blobstore-azure/src/lib.rs

```diff
@@ -23,8 +24,9 @@ use config::StorageConfig;
 
 mod config;
 
-mod bindings {
+pub mod bindings {
```

Comment:
Note, these need to be made public so that they can be used by the integration tests to [mimic starting the provider](https://github.com/wasmCloud/wasmCloud/pull/2993/files#diff-7f900ca14ffd60e466d344ec5b793e01facbf22068206803b9e0e01c1e784beaR204-R218)